Form-Controls/index.html

```diff
@@ -1,28 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+
+    <main>
+      <form>
+        <!-- write your html here-->
+        <!-- try writing out the requirements first-->
+
+        <!-- make sure to take valid input for customer name and email id,
+          let the customer choose any one t-shirt colour by using drop down box,
+          select t-shirt size by using radio button and select delivery date by using 
+          date input box and make sure delivery date is within the next 4 weeks of current date.
+     -->
+
+        <div class="input">
+          <label for="name">Customer Name</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            minlength="3"
+            maxlength="40"
+          />
+        </div>
+        <div class="input">
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" required />
+        </div>
+        <div class="input">
+          <label for="tshirtColor">T-shirt Color</label>
+          <select name="tshirtColor" id="tshirtColor" required>
+            <option value="Black">Black</option>
```

Comment:
There is an approach to not select a default value from the list so that the user is always required to explicitly select an item from the list. This way, we can ensure the user is aware of the choices.

Reply:
@cjyuan  yes, we can achieve it by using a placeholder option like' select a color' so that the user can explicitly select the color.